doc/getting_started.rst

```diff
@@ -1,19 +1,34 @@
 Getting Started with easi
 =========================
 
-easi is a header-only library written in C++11.
-Hence, one simply needs 
-(Only unit tests must be compiled with CMake.)
+easi is a library written in C++14.
+It needs to be compiled with Cmake.
 
 Dependencies
 ------------
 
 Easi depends on the following two projects:
 
--  `ImpalaJIT <https://github.com/uphoffc/ImpalaJIT>`__
 -  `yaml-cpp <https://github.com/jbeder/yaml-cpp>`__
+-  `ASAGI <https://github.com/TUM-I5/ASAGI/>`__
+-  `ImpalaJIT <https://github.com/uphoffc/ImpalaJIT>`__
+
+Only yaml-cpp (version 0.6.x) is a required dependency.
+It can be obtained, for example, from most the package repository of most distributions.
```

Comment:
`from the package manager of most distributions or e.g. as module on SuperMUC-NG.`